node.gyp

```diff
@@ -746,6 +748,47 @@
   ], # end targets
 
   'conditions': [
+    [ 'node_target_type=="static_library"', {
+      'targets': [
+        {
+          'target_name': 'static_node',
+          'type': 'executable',
+          'product_name': '<(node_core_target_name)',
+          'dependencies': [
+            '<(node_core_target_name)',
+          ],
+          'sources+': [
+            'src/node_main.cc',
+          ],
+          'include_dirs': [
+            'deps/v8/include',
+          ],
+          'xcode_settings': {
+            'OTHER_LDFLAGS': [
+              '-Wl,-force_load,<(PRODUCT_DIR)/libnode.a',
+            ],
+          },
+          'conditions': [
+            ['OS in "linux freebsd openbsd solaris android aix"', {
+              'ldflags': [
+                '-Wl,--whole-archive,'
+                '<(OBJ_DIR)/libnode.a '
+                '-Wl,--no-whole-archive',
+              ],
+            }],
+            ['OS=="win"', {
```

Comment:
AFAIK this doesn't need a conditional - `msvs_settings` only works on windows.

Reply:
Good point, I'll remove that condition.